src/bin/install.rs

```diff
@@ -0,0 +1,89 @@
+use cargo::ops;
+use cargo::util::{CliResult, CliError, Config};
+use std::path::Path;
+
+#[allow(dead_code)] // for now until all options are implemented
+
+#[derive(RustcDecodable)]
+struct Options {
+    flag_jobs: Option<u32>,
+    flag_features: Vec<String>,
+    flag_no_default_features: bool,
+    flag_debug: bool,
+    flag_bin: Option<String>,
+    flag_example: Vec<String>,
+    flag_package: Vec<String>,
+    flag_verbose: bool,
+    flag_root: Option<String>,
+}
+
+pub const USAGE: &'static str = "
+Install a crate onto the local system
+
+Installing new crates:
+    cargo install [options]
+    cargo install [options] [-p CRATE | --package CRATE] [--vers VERS]
+    cargo install [options] --git URL [--branch BRANCH | --tag TAG | --rev SHA]
```

Comment:
I think you'll also want to add `flag_git`/`flag_branch`  etc for these sorts of options to the `Options` structure above